wger/manager/templates/workout/view.html

```diff
@@ -145,7 +145,7 @@ <h4 class="modal-title">{% translate "Download as PDF" %}</h4>
     {% if workout.canonical_representation.day_list %}
         <h4>{% translate "Logs" %}</h4>
         <p>
-            <a href="{% url 'manager:log:log' workout.id %}">{% translate 'View logs' %}</a>
+            <button type="button" class="btn btn-primary btn-sm" onclick="window.location.href='{% url 'manager:log:log' workout.id %}'">{% translate 'View logs' %}</button>
```

Comment:
Can you use better something like `<a class="btn btn-primary" href="..." role="button">Foo</a>`? Then things like middle click etc. will keep working

Reply:
Ok I changed it back to a link that looks like a button